src/config/settings.ts

```diff
@@ -374,6 +374,11 @@ const settings: Settings = {
     resolveStatusTimeout: parseInt(process.env.RESOLVE_STATUS_TIMEOUT || '5000'),
   },
   api: {
+    sovryn: {
```

Comment:
let's use it in this way https://github.com/umbrella-network/pegasus/pull/351/files#diff-6833e5b2e60fe1db0c417229f1bdb23109422b3174c37beb31837e86e242d939R429

and not under `api`, let's create `dexes` key in settings, MAtheus usses `pools` in his PR but I will ask him to use `dexes` as well

so: dexes.chainId.protocol